dataactcore/scripts/updateCGAC.py

```diff
@@ -20,21 +21,44 @@ def updateCGAC():
     all_users = userDb.session.query(User).all()
     all_submissions = jobDb.session.query(Submission).all()
 
+    users_to_update = []
+
     for user in all_users:
-        user.cgac_code = findCGACCode(all_cgac, user.cgac_code)
+        if user.cgac_code != None:
+            users_to_update.append(user)
 
-    for submission in all_submissions:
-        submission.cgac_code = findCGACCode(all_cgac, submission.cgac_code)
+    for user in users_to_update:
+        if user.user_id == 2:
```

Comment:
Yup, thanks for catching that. i think I'm going to just remove this script from the repo since it's not a permanent solution. it should've only happened for older users, and everyone except for Booz and Kearney people have been updated.